<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leftDialog</class>
 <widget class="QDialog" name="leftDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>left dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="title">
    <string>   zoom option</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>75</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="zoom_xy">
       <property name="text">
        <string>X and Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="zoom_x">
       <property name="text">
        <string>X only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="zoom_y">
       <property name="text">
        <string>Y only</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="inButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="hello"/>
    </property>
    <property name="text">
     <string>zoom in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>zoom out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>shift up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>shift down</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_xsym">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>K slices - 
X symetry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_createTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>create table</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_itable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>570</y>
     <width>46</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_jtable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>620</y>
     <width>46</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_ktable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>670</y>
     <width>46</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>I-index</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>J-index</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>K-index</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>720</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="table_fname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>760</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>table.csv</string>
   </property>
  </widget>
  <widget class="QPushButton" name="biginButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bigupButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bigdownButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bigoutButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
